src/lib/Components/Inbox/Conversations/Messages.tsx

```diff
@@ -31,6 +32,8 @@ const LoadingIndicator = styled.ActivityIndicator`
 `
 
 export class Messages extends React.Component<Props, State> {
+  flatList: FlatList<any> | any
```

Comment:
This, however, is because our `@types/react-native` are slightly out-of-date:

types/react-native/index.d.ts:
```
/**
     * Scrolls to the item at a the specified index such that it is positioned in the viewable area
     * such that `viewPosition` 0 places it at the top, 1 at the bottom, and 0.5 centered in the middle.
     * May be janky without `getItemLayout` prop.
     */
    scrollToIndex: (params: {animated?: boolean, index: number, viewPosition?: number}) => void
```

[React Native docs](https://facebook.github.io/react-native/docs/flatlist.html#scrolltoindex)
```
Valid params keys are:

'animated' (boolean) - Whether the list should do an animation while scrolling. Defaults to true.
'index' (number) - The index to scroll to. Required.
'viewOffset' (number) - A fixed number of pixels to offset the final target position. Required.
'viewPosition' (number) - A value of 0 places the item specified by index at the top, 1 at the bottom, and 0.5 centered in the middle.
```

Reply:
Is that because `viewOffset` is missing in the typings? If so, send a PR to update https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react-native/index.d.ts

Reply:
https://github.com/DefinitelyTyped/DefinitelyTyped/pull/23648